packages/oruga/src/components/datepicker/Datepicker.vue

```diff
@@ -854,11 +854,11 @@ function onChange(value: string): void {
     const date = (props.dateParser as any)(value, defaultDateParser);
 
     if (
-        date &&
-        Array.isArray(date) &&
-        date.length === 2 &&
-        !isNaN(date[0]) &&
-        !isNaN(date[1])
+        date instanceof Date ||
```

Comment:
Pls change this line to: `(date instanceof Date && !isNaN(date.getTime()))`

Reply:
Fixed [here](https://github.com/oruga-ui/oruga/pull/882/commits/4c1de19279d59c3a9be7188337454221c38fc803).